GSA/digital-gov-drupal
# DIGITAL-150: Adding menu pre role access

## Jira ticket




[DIGITAL-150](https://cm-jira.usa.gov/browse/DIGITAL-150)

## Purpose



This Pr is setting up the ability to limit access per role as far as menu editing ability.

## Includes the following PRs that must be merged first



n/a

## Deployment and testing

### Local Setup



run `lando rebuild -y`

### QA/Testing instructions

1. run `lando drush uli` and impersonate a content admin role. 
2. verify that this role has the ability to edit the footer and main menu  

## Checklist for the Developer




## Checklist for the Peer Reviewers
